docs/platform/nodebalancer/getting-started-with-nodebalancers-new-manager/index.md

```diff
@@ -0,0 +1,119 @@
+
+Nearly all applications that are built using Linodes can benefit from load balancing, and load balancing itself is the key to expanding an application to larger numbers of users. Linode now provides NodeBalancers, which can ease the deployment and administration of a load balancer.
+
+NodeBalancers are built for high availability and designed to be "set and forgotten". The most difficult part of transitioning to a NodeBalancer is simply making sure your application works well in the new environment. In this guide, we will examine a common use case: using a NodeBalancer to load balance a website and prepare it for scaling to thousands of users.
+
+This guide provides a high-level overview of transitioning to a NodeBalancer, but it's outside this page's scope to explain each application a NodeBalancer could balance. For more information on various applications that might be useful behind NodeBalancer, see the rest of Linode Guides & Tutorials.
+
+## Overview
+
+A NodeBalancer listens on a public IP address for incoming connections, then uses configurable rules to select a backend node (out of one or more) to which to send the connection. In HTTP mode, NodeBalancers can reach into the incoming web request and make decisions based on it.
+
+A NodeBalancer allows the incoming load to your application to be spread over any number of Linodes. This, however, comes with new challenges: many web applications have sessions that do not exist outside of the backend node. For this reason, NodeBalancers allow you to specify that the same client should land on the same backend Linode.
+
+As an example, this guide will start with a typical, simple web application.
+
+![The web application when the guide starts.](806-starting-point.png)
+
+There are a few drawbacks to making this application scale, namely that any modifications require updating DNS (which takes time) or inflicting users with downtime. NodeBalancers can bring administrative ease to this application by becoming its front face.
+
+![Diagram after putting NodeBalancer in charge of the web application.](807-first-step.png)
+
+Now, backend Linodes can be added or removed to the work load seamlessly, without end users noticing any downtime.
+
+![Diagram after adding a second web server node.](808-scaling.png)
+
+Additionally, NodeBalancer will watch each backend for failure, rerouting users to working backends transparently and without any manual intervention.
+
+## Configuring a NodeBalancer
+
+Sticking with the simple web application example above, the backend Linode currently powering it must have a private IP address. If it does not, add one now and configure the Linode for [static networking](/docs/networking/configuring-static-ip-interfaces/) -- this will also be a good opportunity to do the same for the database server, since private network traffic is unbilled. Reboot if necessary, and configure your web application to respond on the private IP address, if it is not already.
```

Comment:
Static networking setup isn't really necessary, at least as far as the API/Manager are concerned. If it offers a benefit here, please explain to the readers.